frontend/src/utils.js

```diff
@@ -57,3 +57,10 @@ export const getEditorState = (name) => {
   const contentState = ContentState.createFromBlockArray(contentBlocks, entityMap);
   return EditorState.createWithContent(contentState);
 };
+
+export const getDistinctSortedArray = (arr, property) => {
```

Comment:
More of a flag for discussion - I wouldn't necessarily expect to pass an array of objects in here specifically. I would think with a name like that it would be just de-duping and sorting.

Reply:
I have moved the map outside of the utils function.